app/tests/hold-tap/tap-unless-interrupted/6-global-quick-tap/1-basic/native_posix_64.keymap

```diff
@@ -6,17 +6,17 @@
 &kscan {
 	events = <
 		/* tap */
-		ZMK_MOCK_PRESS(0,0,10) 
-		ZMK_MOCK_RELEASE(0,0,10) 
-		/* normal quick tap */
+		ZMK_MOCK_PRESS(0,0,10)
+		ZMK_MOCK_RELEASE(0,0,250)
+		/* quick tap */
 		ZMK_MOCK_PRESS(0,0,400)
 		ZMK_MOCK_RELEASE(0,0,400)
 		/* hold */
 		ZMK_MOCK_PRESS(0,0,400)
 		ZMK_MOCK_PRESS(1,0,10)
 		ZMK_MOCK_RELEASE(1,0,10)
 		ZMK_MOCK_RELEASE(0,0,400)
-		/* global quick tap */
+		/* min prior term */
```

Comment:
I don't understand this comment - maybe keep the old comment?

Reply:
Oops, that's a remnant from when I had renamed `global_quick_tap` to `min_prior_ms`, good catch! I'll clean that up when I write the docs.